config.ru

```diff
@@ -5,3 +5,8 @@ require_relative "loader"
 clover_freeze
 
 run(Config.development? ? Unreloader : Clover.freeze.app)
+
+unless Config.development?
+  require "tilt/sass" unless File.exist?(File.expand_path("../compiled_assets.json", __FILE__))
+  Tilt.finalize!
+end
```

Comment:
@enescakir do we use sass at all? wondering if we should drop this branch entirely.

Reply:
We don't use sass

Reply:
I'll delete tomorrow, then. Well, later today.